### ISSUE TITLE
Publish latest version to the OperatorHub

### ISSUE THREAD

**Issue by lsoica** [OT-CONTAINER-KIT/redis-operator#979]
The latest version available in the OperatorHub for the Redis Operator is 0.15.1.

**Comment by drivebyer** [comment]
There is an alternative image: `docker pull ghcr.io/ot-container-kit/redis-operator/redis-operator:v0.17.0`.

**Comment by lsoica** [comment]
> There is an alternative image: `docker pull ghcr.io/ot-container-kit/redis-operator/redis-operator:v0.17.0`.

That's true, but the latest version listed under operatorhub is 0.15.1 https://operatorhub.io/operator/redis-operator
And as far as I know, that's because this is the last version that was published under https://github.com/redhat-openshift-ecosystem/community-operators-prod/tree/main/operators/redis-operator

This repository is the source or truth for not only the operatorhub, but also for an operator catalog that's available with any OKD/OCP cluster. Without publishing new versions under that repository, we have to bring our own operator to an OKD/OCP cluster instead of consuming what's available out of the box in the catalog.

**Comment by sedadas** [comment]
This has happened many times previously where new releases were not published to operatorhub.
See #245 #517 
Perhaps an automated pipeline can be developed to do this.

**Comment by Elyytscha** [comment]
also was wondering why i could only install version 0.15.1 on okd 4.16

![image](https://github.com/user-attachments/assets/fe46765c-87cb-4219-8677-f5f0a47dd8ed)

**Comment by ctml91** [comment]
also waiting for this. 0.15.1 package is bugged and missing RBAC for sentinel/replication CRD's.

**Comment by drivebyer** [comment]
The latest image has been pushed https://quay.io/repository/opstree/redis-operator?tab=tags

**Comment by ctml91** [comment]
> The latest image has been pushed https://quay.io/repository/opstree/redis-operator?tab=tags

0.15.1 on OCP & k8s OperatorHub
https://operatorhub.io/operator/redis-operator

**Comment by drivebyer** [comment]
> 0.15.1 on OCP & k8s OperatorHub
> https://operatorhub.io/operator/redis-operator

I work on this repo in my spare time. Nice PRs are welcome!

**Comment by bulsiel** [comment]
Any progress in that matter? This issue is quite important for OpenShift and opratorhub users? 
@drivebyer  - is there any chance for resolving this issue? Thank you for your interest in this matter.

**Comment by drivebyer** [comment]
PTAL @shubham-cmyk

**Comment by shubham-cmyk** [comment]
I have to ping @iamabhishek-dubey for this

**Comment by bulsiel** [comment]
Hi,
Any news? 
@shubham-cmyk , @iamabhishek-dubey

**Comment by DodoLeDev** [comment]
For now, you can work around this issue with OLM by editing the corresponding ClusterServiceVersion and changing the following values:
- `metadata.annotations.containerImage`: `quay.io/opstree/redis-operator:v0.21.0`
- `install.spec.deployments[0].spec.template.spec.containers[0].image`: `quay.io/opstree/redis-operator:v0.21.0`
- `install.spec.deployments[0].spec.template.spec.containers[0].args`: `['manager', '--leader-elect']`
- `install.spec.deployments[0].spec.template.spec.containers[0].command`: `['/operator']`

**Comment by iamabhishek-dubey** [comment]
I will get this release by tomorrow

**Comment by ibotty** [comment]
Anything holding it back?  Is there a PR which we could push over the finishing line?

**Comment by sridharkothandan** [comment]
@iamabhishek-dubey  Hi Mate, is it possible to the get version .22 released to the operator Hub, as Openshift uses it from there.

**Comment by sridharkothandan** [comment]
Hi @iamabhishek-dubey    i created the operator versions based on git tag for all missing versions. https://github.com/sridharkothandan/redis-operator/tree/main/temp-community-operators/operators/redis-operator

please review and Submit them to Operator Hub

**Comment by sridharkothandan** [comment]
@iamabhishek-dubey  https://github.com/k8s-operatorhub/community-operators/pull/6943

**Comment by finomen** [comment]
> [@iamabhishek-dubey](https://github.com/iamabhishek-dubey) [k8s-operatorhub/community-operators#6943](https://github.com/k8s-operatorhub/community-operators/pull/6943)

Is there any progress on it?

**Comment by btw02** [comment]
Hi, I wanted to follow up on this request.

The latest version as os 13.04.2026 is `v0.24.0`, while on Operator Hub there is still only `v0.15.1` available:
I would be very grateful, if you could update the version @iamabhishek-dubey @deepanshumishra 

<img width="1776" height="1860" alt="Image" src="https://github.com/user-attachments/assets/eb39ed05-97c9-494e-a7bf-3fefe6362ecc" />

**Comment by serena-cortopassi** [comment]
Any news? I'll test the following but I'll prefer to have the latest version available from the Hub directly. Thanks

> For now, you can work around this issue with OLM by editing the corresponding ClusterServiceVersion and changing the following values:
> 
> * `metadata.annotations.containerImage`: `quay.io/opstree/redis-operator:v0.21.0`
> * `install.spec.deployments[0].spec.template.spec.containers[0].image`: `quay.io/opstree/redis-operator:v0.21.0`
> * `install.spec.deployments[0].spec.template.spec.containers[0].args`: `['manager', '--leader-elect']`
> * `install.spec.deployments[0].spec.template.spec.containers[0].command`: `['/operator']`

**Comment by alita1991** [comment]
Hi @drivebyer @iamabhishek-dubey, any news on this request? If you need any help, just tell me on which repo I have to make the changes to publish the latest version, and we'll help.

**Comment by iamabhishek-dubey** [comment]
This should be done by today

**Comment by kriss-s** [comment]
would it be possible to keep updating the RedHat community operator Hub as well as OperatorHub.io?

Like done in the past https://github.com/redhat-openshift-ecosystem/community-operators-prod/pulls?q=is%3Apr+redis+is%3Aclosed

**Comment by cmays20** [comment]
Would love to see this operator get updated in the Red Hat openshift ecosystem.

**Comment by ingvaldlorentzen** [comment]
Same here, really want to use this operator through OpenShift OperatorHub!

**Comment by TrymaJohnsen** [comment]
@iamabhishek-dubey @drivebyer :)
